### ISSUE TITLE
Password is stored in plain text

### ISSUE THREAD

**Issue by nosmo** [equalitie/RVD-webapp#3]
:hankey:

**Comment by nosmo** [comment]
(sorry, not a very constructive ticket - I just couldn't think of anything to put in the body)

**Comment by l-r** [comment]
yeah, @redwire and I had to cut a bunch of corners last night to make the deadline, I've a short list of things that need to be fixed for production use, which I'll turn into tickets when I have a minute :)

**Comment by l-r** [comment]
fixed in https://github.com/equalitie/RVD-webapp/commit/a105002c0f66c761449a84940e1e365eb3a971f2
